### DISCUSSION TITLE
[@types/three] Three.js missing type definitions on OrbitControl.d.ts

### DISCUSSION THREAD

**Discussion by LauferAlex** [DefinitelyTyped/DefinitelyTyped#58090] [Issues with a @types package]
So, I’m using THREE.js library on my angular application on which I’m using the OrbitCotrol example. 

On the official documentation, there is a couple of properties that I need to access but are not declared as module definition.
![image](https://user-images.githubusercontent.com/86115165/148793817-56041ce6-2056-4c0f-805d-f366dddb44c0.png)

The properties that are not declared are:
- .target0
- .position0
- .zoom0

I would like to know if someone could help me approve this small fix, I have a really urgent project with short deadline. So I would really appreciate a quick answer.

Thank you.

[a comment by an automated account is not shown]

**Comment by LauferAlex** [comment]
I can generate the pull request, but a need to be a contribuitor in order to access the code.
Please anyone can help me with that?

**Reply by Methuselah96** [reply]
You should make the pull request on [three-ts-types](https://github.com/three-types/three-ts-types). See the [contributing guidelines](https://github.com/three-types/three-ts-types/blob/master/CONTRIBUTING.md) for how to get started.

**Reply by LauferAlex** [reply]
Thank you for letting me know. I just made the pull request. 
PR Link: https://github.com/three-types/three-ts-types/pull/157
